public/css/styles.css

```diff
@@ -0,0 +1,673 @@
+/*---general page and components css---*/
+html {
+  box-sizing: border-box;
+}
+
+*, *:before, *:after {
+  box-sizing: border-box;
+}
+
+* {
+  overflow: auto;
+}
+
+body {
+  width: 100%;
+  margin: auto;
+  font-family: Helvetica, Arial, "Lucida Grande", sans-serif;
+  font-weight: 300;
+}
+
+nav {
+  margin-left: -1em;
```

Comment:
I don't see what benefit this has. As a general rule I consider negative margins to be an anti-pattern. If you're going to use a negative margin, be sure you actually need it.